its/plugin/tests/src/test/java/com/sonar/it/java/suite/SuppressWarningTest.java

```diff
@@ -51,6 +51,13 @@ public void suppressWarnings_nosonar() throws Exception {
     assertThat(parseInt(getMeasure(projectKey, "violations").getValue())).isEqualTo(5);
   }
 
+  // NOTE: This test is currently disabled due to SONARJAVA-4553.
+  // Even when SONARJAVA-4553 is fixed, we cannot simply re-enable the test, as it will fail for other reasons:
+  //   - PMD does not seem to support Java 17 (to be confirmed)
+  //   - The PMD plugin is not compatible with on-demand plugin downloading
```

Comment:
Indeed and here is the ticket we created to inform sonar-pmd: https://github.com/jborgers/sonar-pmd/issues/421